notebooks/README.rst

```diff
@@ -8,6 +8,7 @@ create a Python 3 virtual environment, and install the dependencies from
 .. code::
 
    $ pipenv shell
+   (pipenv) $ pip install ..
```

Comment:
`pip install -e ..` is slightly better.

Reply:
`-e` pins the `robust` package to its source directory. That means that if you make any changes to the source, they'll automatically be reflected within Python. Using the standard way of installing packages, those changes would be not reflected until you manually reinstall the package via `pip install ..`